<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ImageWidget</class>
    <widget class="QWidget" name="ImageWidget">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>960</width>
                <height>640</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Image Loader</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="4,1">
            <property name="spacing">
                <number>0</number>
            </property>
            <property name="leftMargin">
                <number>0</number>
            </property>
            <property name="topMargin">
                <number>0</number>
            </property>
            <property name="rightMargin">
                <number>0</number>
            </property>
            <property name="bottomMargin">
                <number>0</number>
            </property>
            <item>
                <widget class="QLabel" name="display_label">
                    <property name="text">
                        <string/>
                    </property>
                    <property name="alignment">
                        <set>Qt::AlignCenter</set>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QHBoxLayout" name="img_layout" stretch="1,2,1,1">
                    <property name="spacing">
                        <number>0</number>
                    </property>
                    <property name="leftMargin">
                        <number>0</number>
                    </property>
                    <property name="topMargin">
                        <number>0</number>
                    </property>
                    <property name="rightMargin">
                        <number>0</number>
                    </property>
                    <property name="bottomMargin">
                        <number>0</number>
                    </property>
                    <item>
                        <widget class="QToolButton" name="ocr_btn">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="styleSheet">
                                <string notr="true">background-color: rgb(92, 53, 102);
                                    color: rgb(255, 255, 255);
                                </string>
                            </property>
                            <property name="text">
                                <string>OCR</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QToolButton" name="load_btn">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="styleSheet">
                                <string notr="true">background-color: rgb(32, 74, 135);
                                    color: rgb(255, 255, 255);
                                </string>
                            </property>
                            <property name="text">
                                <string>load from file</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QToolButton" name="bin_btn">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="styleSheet">
                                <string notr="true">background-color: rgb(143, 89, 2);
                                    color: rgb(255, 255, 255);
                                </string>
                            </property>
                            <property name="text">
                                <string>binaryzation</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QToolButton" name="reset_btn">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="styleSheet">
                                <string notr="true">background-color: rgb(78, 154, 6);
                                    color: rgb(255, 255, 255);
                                </string>
                            </property>
                            <property name="text">
                                <string>reset</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <resources>
        <include location="../../resources/app.qrc"/>
    </resources>
    <connections/>
</ui>
